### ISSUE TITLE
Provider for users from Russia, Ukraine...

### ISSUE THREAD

**Issue by razer1s** [mancuniancol/repository.providers.quasar#4]
Hi. I from Ukraine and use Quasar, but no any provider for Quasar that searches film and tv shows (torrents for it) on Russian language. You can create someone for ruhunt.org. 
I create simple provider due to example from github, but it doesn't search shows, without settings(quality filter) and doesn't show size of torrents(mean video file). I'll be very thankful... sorry for my English

**Comment by mancuniancol** [comment]
Of course, I can help you.  Please, could you check the first version?
I've turned off all the filters  https://github.com/mancuniancol/script.quasar.ruhunt-mc/archive/v1.0.0a.zip

I will ask you to help me with the translation to russian.

[strings.zip](https://github.com/mancuniancol/repository.providers.quasar/files/121489/strings.zip)

You can edit the file with Poedit software.  https://poedit.net/download

Thank you

**Comment by razer1s** [comment]
Finished translation.
https://yadi.sk/i/BwmmbCA4oWWaS
select "Скачать"

**Comment by mancuniancol** [comment]
OK. ;)  I will add it to the providers, then it will be Russian interface for everybody

**Comment by razer1s** [comment]
refresh translation
